Price list small screen

lomamech opened the pull request:
- Update html of price list to keep all functionality in Small Screen
- Update function of translation calling

Closes #747 .

lomamech committed:
Update html of price list to keep all functionality in Small Screen

lomamech committed:
Update function of translation calling

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -1,27 +1,28 @@
-      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
-      <li class="title">{{ "PRICE_LIST.TITLE" | translate }}</li>
+      <li class="static" translate> TREE.ADMIN </li>
+      <li class="active" translate> PRICE_LIST.TITLE </li>
```
This class should be "title".

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -1,27 +1,28 @@
+   <div class="toolbar">
+     <div class="toolbar-item">
+      <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
+        <span class="fa fa-plus"></span><span translate> {{ "PRICE_LIST.NEW_PRICE_LIST" | translate }} </span>
```
If you could put a space between the `</span> <span translate>`, it would look nicer.

Also, because you have the `translate` directive, you don't need the filter.  You can just do this:
```html
<span class="fa fa-plus"></span><span translate>PRICE_LIST.NEW_PRICE_LIST</span>
```

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -1,27 +1,28 @@
-  </button>
+   <div class="toolbar">
+     <div class="toolbar-item">
+      <button class="btn btn-sm btn-default" id="create" ng-click="PriceListCtrl.create()" data-method="create">
```
Most of our buttons have been normal buttons.  Could you remove the `btn-sm` class to make it look the same as other modules?

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -1,27 +1,28 @@
-          <div class="panel-heading">
-            {{ "TABLE.COLUMNS.LISTS" | translate }}
+      <div class="col-md-6">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
```
This is a big improvement :+1:

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -41,40 +42,41 @@
             <ul style="list-style-type: none; padding-left: 15px;">
-              <li><i class="glyphicon glyphicon-pencil"></i> {{ "PRICE_LIST.HELP_TXT_2" | translate }}</li>
-              <li><i class="glyphicon glyphicon-trash"></i> {{ "PRICE_LIST.HELP_TXT_3" | translate }}</li>
+              <li><i class="glyphicon glyphicon-pencil"></i><span translate> PRICE_LIST.HELP_TXT_2 </span> </li>
```
In the future, we should swap these glyphicons out with font awesome.  That can be another PR though.

jniles commented on client/src/partials/price_list/pricelist.html:
```diff
@@ -146,18 +149,16 @@
+                    <th translate> TABLE.COLUMNS.VALUE</th>
+                    <th translate> TABLE.COLUMNS.PERCENT </th>
+                    <th translate> TABLE.COLUMNS.INVENTORY </th>
+                    <th colspan="6" translate> TABLE.COLUMNS.ACTION </th>
```
This `colspan="6"` looks wrong.   Is it really necessary?  Does the module actually use it?

jniles reviewed (commented):
@lomamech, this module now behaves correctly!  Well done.

I've made a few comments where the code could be improved or typos prevent the benefits of your changes.  Otherwise, this looks really good.

lomamech committed:
Sanitaza code
Using the common class for button
using font awesome instead glyphicons
Using not necessary colspan

jniles reviewed (approved):
LGTM!
